### PR TITLE
update wrapping of applyRemesh, fix #1930

### LINKED ISSUE #1930: Update applyRemesh function in the python wrapping of Feelpp
The version of the function `applyRemesh` in the python wrapping is outdated

### PR DESCRIPTION
**Pull request by lberti**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes?
- [ ] Have you successfully run the Feel++ testsuite with your changes locally?
- [ ] Have you written Doxygen comments in your contribution ?

-----

### CODE REVIEW

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp [hunk 1]
```diff
@@ -88,7 +88,12 @@ void defFM(py::module &m)
         .def("exportResults",static_cast<void (fm_t::*)( double )>(&fm_t::exportResults), "export the results of the fluid mechanics problem", py::arg("time"))
 
         // remesh
-        .def("applyRemesh",&fm_t::applyRemesh, "apply remesh to toolbox and regenerate the necessary data structure")
+        .def("applyRemesh",
```
Comment (prudhomm) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp:91]: @lberti please add `py::arg` for each parameters

**Review by prudhomm — APPROVED** [review]

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp (hunk 1 above)
Comment (lberti) [reply python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp:91]: Done

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp [hunk 2]
```diff
@@ -93,7 +93,7 @@ void defFM(py::module &m)
         {
             std::shared_ptr<RemeshInterpolation> remeshInterp = std::make_shared<RemeshInterpolation>();
             self->applyRemesh(meshOld,meshNew,remeshInterp);            
-        }, "apply remesh to toolbox and regenerate the necessary data structure")
+        }, "apply remesh to toolbox and regenerate the necessary data structure",py::arg("oldMesh"),py::arg("newMesh"))
```
Comment (prudhomm) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp:96]: @lberti could you rename to "from" and "to" or "old" and "new" ?

**Review by prudhomm — CHANGES_REQUESTED** [review]

**Comment by prudhomm** [comment]
also update the branch when you push the change to be in sync with `develolp`

**Review by prudhomm — APPROVED** [review]

**Pull request closed without merging**